.buildkite/browser-pipeline.yml

```diff
@@ -16,228 +16,206 @@ steps:
       - docker-compose#v3.9.0:
           push:
             - browser-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-browser-${BRANCH_NAME}
-#
-  - label: ':chrome: v30 Browser tests'
-    depends_on: "browser-maze-runner-image"
-    timeout_in_minutes: 20
-    plugins:
-      docker-compose#v3.9.0:
-        pull: browser-maze-runner
-        run: browser-maze-runner
-        use-aliases: true
-        verbose: true
-        command:
-          - --farm=bs
-          - --browser=chrome_30
-    concurrency: 5
-    concurrency_group: 'browserstack'
 
-  - label: ':chrome: v43 Browser tests'
-    depends_on: "browser-maze-runner-image"
+  - label:  ':docker: Build browser maze runner v6 image'
+    key: "browser-maze-runner-v6-image"
     timeout_in_minutes: 20
     plugins:
-      docker-compose#v3.9.0:
-        pull: browser-maze-runner
-        run: browser-maze-runner
-        use-aliases: true
-        verbose: true
-        command:
-          - --farm=bs
-          - --browser=chrome_43
-    concurrency: 2
-    concurrency_group: 'browserstack'
+      - artifacts#v1.5.0:
+          download: min_packages.tar
+          build: ${BUILDKITE_TRIGGERED_FROM_BUILD_ID}
+      - docker-compose#v3.9.0:
+          build:
+            - browser-maze-runner-v6
+          image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
+          cache-from:
+            - browser-maze-runner-v6:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-browser-v6-${BRANCH_NAME}
+      - docker-compose#v3.9.0:
+          push:
+            - browser-maze-runner-v6:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-browser-v6-${BRANCH_NAME}
 
-  - label: ':chrome: v61 Browser tests'
+# BitBar tests
+
+  - label: ':firefox: 101 Browser tests'
```

Comment:
Could we somehow make it obvious on the Buildkite build page which steps were run on BB and which were run on BS?  Could with maybe prefix the label of each step with either "BB -" or "BS -"?  Or maybe Buildkite annotations could be used here, like providing a header for each set of steps (I've never tried this, so don't how how it would actually look in practice).